base/acme/src/org/dogtagpki/acme/ACMEOrder.java

```diff
@@ -64,7 +70,25 @@ public Date getExpirationTime() {
 
     public void setExpirationTime(Date expirationTime) {
         this.expirationTime = expirationTime;
-        expires = ACME.DATE_FORMAT.format(expirationTime);
+        expires = expirationTime == null ? null : ACME.DATE_FORMAT.format(expirationTime);
```

Comment:
Ah my friend `ACME.DATE_FORMAT` strikes again!

Reply:
Yeah, just adding a guard for NPE.